modules/core/client/components/LanguageSwitchPresentational.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown, MenuItem } from 'react-bootstrap';
+
+/**
+ * A presentational component of LanguageSwitch.
+ * Is used in header for non-logged users.
+ * @param {String} currentLanguageCode - code of current language
+ * @param {Array} languages - all the languages in selection
+ * @param {Function} onChangeLanguage - what to do when language is changed
+ */
+export function LanguageSwitchDropdown({ currentLanguageCode, languages, onChangeLanguage }) {
+  // languages which are not selected
+  const otherLanguages = languages.filter(language => language.code !== currentLanguageCode);
+  const currentLanguage = languages.find(language => language.code === currentLanguageCode);
+
+  return (
+    <Dropdown
+      bsSize="large"
+      title={currentLanguage.name}
+      id={`dropdown-basic-${currentLanguage.code}`}
+      pullRight
+      className="disable-hover-display"
+    >
+      <Dropdown.Toggle className="btn-inverse">
+        {currentLanguage.name}
+      </Dropdown.Toggle>
+      <Dropdown.Menu>
+        {otherLanguages.map(language => (
```

Comment:
I think we could simplify this and list all the languages regardless, and just mark list item that matches `currentLanguage.code` with `active` -prop and it'll get highlighted.

It'll be double in toggle and in the list but it's okay, user can orient themself easier with list where items don't dissapear. (and code is simpler, yay!)

Reply:
247f92c88dce87906516c9a1521b2e624ab94d06

![selector](https://user-images.githubusercontent.com/7449720/53290416-e2749980-37a3-11e9-949c-ee6946a7dd77.png)